spec/scenic/adapters/postgres_spec.rb

```diff
@@ -26,6 +26,14 @@ module Adapters
           expect(view.name).to eq("greetings")
           expect(view.materialized).to eq true
         end
+
+        it "raises an exception if the version of PostgreSQL is too old" do
+          adapter = Postgres.new
+          allow(adapter).to receive(:version_number) { 920 }
+
+          expect { adapter.create_materialized_view('greetings', 'select 1') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.